src/svcutilcore/pkg/svcutilcore.pkgproj

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- Skip package validation since this package will now contain the shims and this is not allowed by validation -->
```

Comment:
Looks like this comment should be removed too. Now that we don't have shim, I am wondering if we still need SkipValidatePackage=true below.